frontend/src/components/LogIn.tsx

```diff
@@ -5,7 +5,7 @@ export interface LogInProps {
   buttonClicked: () => void
 }
 
-export function LogIn(props: LogInProps) {
+export function LogIn(props:LogInProps) {
```

Comment:
Nit: it seems like there should be extra space between props and LoginInProps, can you run prettier?

Reply:
No, but did it this time